partner_identification_dea/readme/ROADMAP.rst

```diff
@@ -0,0 +1,2 @@
+This Module is Manage the Medical License, DEA Number and its Expired Date for Contacts
```

Comment:
This is duplicate with the DESCRIPTION.rst. Huse the ROADMAP.rst to announce plans and ideas for the module that might be taken up in the future.